external.d.ts

```diff
@@ -0,0 +1,11 @@
+declare module "universal-user-agent" {
+    function getUserAgentNode(): string
+    export = getUserAgentNode;
+}
+declare module "is-plain-object" {
+    export = isPlainObject;
+
+    function isPlainObject(o: any): boolean;
+
+    namespace isPlainObject { }
```

Comment:
why is the `namespace` required here?

Reply:
I don't think it's needed. 

Probably a remnant from the automatically generated definition from dts-gen